libc-test/build.rs

```diff
@@ -2231,6 +2231,9 @@ fn test_linux(target: &str) {
     let aarch64_musl = target.contains("aarch64") && musl;
     let gnuabihf = target.contains("gnueabihf");
 
+    let supports_unaligned_memory_access =
+        !(mips32 || mips64 || sparc64 || s390x);
```

Comment:
Should we really skip tests on their targets?

Reply:
The problem is that reading from packed fields should take into account that the field is unaligned. I don't think the rust side of the CTest generated code doesn't this, thats why the roundrip tests fail for the packed struct on these targets.

Reply:
So, I think it's better to have different definitions. Did you confirm that items work well on skipped targets?